src/test/java/com/google/sps/servlets/CreateBlobstoreUrlServletTest.java

```diff
@@ -0,0 +1,63 @@
+package com.google.sps;
+
+import com.google.sps.servlets.CreateBlobstoreUrlServlet;
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+import com.google.appengine.api.blobstore.BlobstoreService;
+import com.google.appengine.api.blobstore.BlobstoreServiceFactory;
+import org.junit.Assert;
+import org.junit.runners.JUnit4;
+import org.junit.runner.RunWith;
+import org.junit.Test;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+@RunWith(JUnit4.class)
+public final class CreateBlobstoreUrlServletTest {
+  CreateBlobstoreUrlServlet createBlobstoreUrlServlet = new CreateBlobstoreUrlServlet();
+  
+  public BlobstoreService mockBlobstore(String uploadUrl) {
+    BlobstoreService blobstoreService = mock(BlobstoreService.class);
+    when(blobstoreService.createUploadUrl("/postData")).thenReturn(uploadUrl);
```

Comment:
Since you use "/postData" in multiple places, perhaps a final variable could be made for it?

Reply:
Good catch